### ISSUE TITLE
Error compiling wram.asm

### ISSUE THREAD

**Issue by phramos07** [pret/pokeyellow#28]
Hi! I get an error when compiling the file wram.asm on OSX Yosemite with the newest version of rgbds (installed it with homebrew --HEAD).

```
ERROR: wram.asm(536):
	Constant mustn't be negative: -528
make: *** [wram.o] Error 5
```

On line 536:

`ds wPrinterSerialReceived -@`

I just started venturing into this land of rom hacking. Anyone has any ideas of what may be happening?

**Comment by dannye** [comment]
As stated in the other conversation, either use rgbds 0.2.5 or wait for #27 to be merged. I'll look into merging it now.

**Comment by yenatch** [comment]
https://github.com/pret/pokered/issues/163

**Comment by dannye** [comment]
Fixed by #27
